content/millennium/r4/request-and-response/communication.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Communication | R4 API
+---
+
+# Communication
+
+* TOC
+{:toc}
+
+## Overview
+
+The Communication resource is a conveyance of information from one entity, a sender, to another entity, a receiver. The information includes encoded data and optionally a related Patient and a related Encounter.
+
+The following fields are returned if valued:
+
+* [Communication Id](https://hl7.org/FHIR/communication-definitions.html#Communication.identifier)
+* [Status (Completed)](https://hl7.org/FHIR/communication-definitions.html#Communication.status)
+* [Category](https://hl7.org/FHIR/communication-definitions.html#Communication.category)
+* [Priority](https://hl7.org/FHIR/communication-definitions.html#Communication.priority)
+* [Subject](https://hl7.org/FHIR/communication-definitions.html#Communication.subject)
+* [Topic](https://hl7.org/FHIR/communication-definitions.html#Communication.topic)
+* [Encounter](https://hl7.org/FHIR/communication-definitions.html#Communication.encounter)
+* [Sent DateTime](https://hl7.org/FHIR/communication-definitions.html#Communication.sent)
```

Comment:
The 'In Response To' field needs to be added. https://hl7.org/FHIR/communication-definitions.html#Communication.inResponseTo

Reply:
https://github.com/cerner/fhir.cerner.com/pull/607/commits/99c1be98f1d5fe1d1fa6c1cfa9b11d4c798094b9#diff-cc6586eb639048bdea01208ce786612c64361622cd67bd53a5bf7253f27a840aR17